//any error in the function other than syntax error is bearable untill it invokes
//it stores the function the moment it reads
function check_1(){
    const x=10;
    x=20;
}
console.log('check point 1');//check point 1
function check_2(){
    console.log(s);
}
console.log('check point 2');//check point 2
// function check_3(){
//     let let=30 and some syntax error
// }